test/mocks.ts

```diff
@@ -26,4 +26,35 @@ module Mocks {
       return true;
     }
   }
+
+  export class Scale<D, R> extends Plottable.Scale<D, R> {
```

Comment:
The only alternative to not using a mock of the parent class is to make subsequent tests for each of the subclasses, which is quite infeasible as that would multiply the number of tests by the number of subclasses (This already starts to be infeasible at 2...)